### DISCUSSION TITLE
Inverting settings for VRM

### DISCUSSION THREAD

**Discussion by Voorspelling** [Louisvdw/dbus-serialbattery#256] [Q&A] [answered]
When my batteries (2 times 48v) is charging it says in VRM discharging, and when the battery is discharging to my victron stuff it says charging, can i alter this in the correct settings

**Comment by Louisvdw** [comment, marked as answer]
You most likely have a Daly BMS? They have some models where the data published is swapped. 
So for Daly there is a option to swop this.
Edit utils.py and change it to
INVERT_CURRENT_MEASUREMENT = -1

**Comment by Voorspelling** [comment]
thanx that is fast i will try that when i am at home. i thought so that it would be just one setting.

**Comment by Voorspelling** [comment]
and yes i have 2 daly,s
